inspire_utils/name.py

```diff
@@ -105,7 +105,8 @@ def first(self):
     @property
     def first_initials_list(self):
         names_no_dash_list = itertools.chain.from_iterable(name.split("-") for name in self.first_list)
-        return [(name[0] + u'.') for name in names_no_dash_list if name]
+        names_split_on_dot = itertools.chain.from_iterable([name.split('.') for name in names_no_dash_list])
```

Comment:
You don't need a concrete list, any iterable works:
```suggestion
        names_split_on_dot = itertools.chain.from_iterable(name.split('.') for name in names_no_dash_list)
```